src/libcore/cell.rs

```diff
@@ -1177,12 +1265,27 @@ impl<'b> BorrowRefMut<'b> {
     fn new(borrow: &'b Cell<BorrowFlag>) -> Option<BorrowRefMut<'b>> {
         match borrow.get() {
             UNUSED => {
-                borrow.set(WRITING);
+                borrow.set(MIN_WRITING);
                 Some(BorrowRefMut { borrow: borrow })
```

Comment:
Should this method maybe get a comment that it deliberately doesn't increment, because this is what is called by `RefCell::borrow_mut()` and hence must only succeed if there is no writer at all yet -- unlike `BorrowRefMut::clone()`?

Reply:
Done.